### PR TITLE
Feature/ctsi 73 fetch proof

### PR DESCRIPTION
**Pull request by fabiooshiro**

### CODE REVIEW

File: api/reader.graphql [hunk 1]
```diff
@@ -1,9 +1,25 @@
...
+  inputIndex: Int!
+  lastInput: Int!
+
+  """
+  A globally unique identifier. Can be used in various places throughout the system to identify this single value.
+  """
+  nodeId: ID!
+
+  outputIndex: Int!
+  validityInputIndexWithinEpoch: Int!
```
Comment (sandhilt) [inline api/reader.graphql:16]: Is it flatten? Some reason for that?

**Review by DiegoSena — APPROVED** [review]

File: api/reader.graphql (hunk 1 above)
Comment (fabiooshiro) [reply api/reader.graphql:16]: it is the last proposal from the node group

Comment (fabiooshiro) [reply api/reader.graphql:16]: and probably it's not a big deal because the contracts need this format.

Comment (fabiooshiro) [reply api/reader.graphql:16]: also, may it will change

**Review by sandhilt — APPROVED** [review]
LGTM!

File: api/reader.graphql (hunk 1 above)
Comment (fabiooshiro) [reply api/reader.graphql:16]: https://github.com/cartesi/rollups-node/pull/337/files#diff-7041aafed4cac335c7cfd177729a5063242905c2460c78d40a424e8875a7e450R259

Comment (fabiooshiro) [reply api/reader.graphql:16]: https://discord.com/channels/600597137524391947/1172230296066871407/1230258214625480816

**Pull request merged**